csrc/host_ir/jit.cpp

```diff
@@ -46,11 +46,10 @@ constexpr std::string_view kMainFuncName = "main";
 constexpr std::string_view kTensorSizeFuncName = "tensor_size";
 constexpr std::string_view kAllocateTensorFuncName = "allocate_tensor";
```

Comment:
I'd rename allocate_tensor to new_tensor and deallocate_tensor to delete_tensor. The name `allocate_tensor` gives me the wrong impression that it tries to allocate the underlying storage, but it doesn't. at_empty_strided_cuda does that instead.